### PR TITLE
wordy: add topics for exercise to config.json per #850

### LINKED ISSUE #850: wordy: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

Please comment on this issue or reference it in your pull request (using #issue-number) if you're going to be working on it. If you put "fixes #issue-number" in the pull request description then it will also automatically close this issue when the pull request gets merged :)

### PR DESCRIPTION
**Pull request by m-a-ge**
Fixes #850 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -579,7 +579,12 @@
       "core": false,
       "difficulty": 6,
       "slug": "wordy",
-      "topics": null,
+      "topics": [
+        "strings",
+        "parsing",
+        "pattern_matching",
+        "transforming"
```
Comment (FridaTveit) [inline config.json]: Great suggestions! :) I would maybe add `logic` and `integers` as well. And maybe `regular_expressions`? What do you think?

Comment (FridaTveit) [reply config.json]: And also `exception_handling` since the tests expect an exception to be thrown :)

Comment (m-a-ge) [reply config.json]: Good points, I'll update the PR shortly

**Comment by m-a-ge** [comment]
@FridaTveit I've updated PR based on your suggestions

**Pull request merged**